x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list.tsx

```diff
@@ -52,13 +52,13 @@ export const CreateEngineButton: React.FC<CreateEngineButtonProps> = ({ disabled
 
   return (
     <EuiPopover
-      isOpen={disabled && showPopover}
+      isOpen={showPopover}
       closePopover={() => setShowPopover(false)}
       button={
         <div
           data-test-subj="create-engine-button-hover-target"
           onMouseEnter={() => setShowPopover(true)}
-          onFocus={() => setShowPopover(true)}
+          onMouseLeave={() => setShowPopover(false)}
```

Comment:
we may still want a focus event for users navigating with a keyboard

Reply:
Yes, we should definitely have that. I am also concerned about other potential accessibility issues here. Does the popover announce its presence when opened? Does tabIndex 0 make sense in the context of this page? Does onFocus get triggered when someone tabs to the button, or only when it tabs to the div?

Reply:
I'll get these fixed in a follow up PR. thanks for the feedback